setup.cfg

```diff
@@ -28,18 +28,23 @@ exclude =
 # D400 First line should end with a period
 # D401 First line should be in imperative mood
 # D403 First word of the first line should be properly capitalized
+# DAR101 Missing parameter(s) in Docstring: - min_hits
 # E128 continuation line under-indented for visual indent
 # E251 unexpected spaces around keyword / parameter equals
+# E501 line too long (105 > 100 characters)
```

Comment:
@WenzDaniel , with this one I would disagree, it would be nice if we can try and limit this line length. Otherwise we can increase the value to be a bit higher using the 100 a few lines above.

Reply:
Nevertheless, I think 100 is a good rule of thumb

Reply:
otherwise 120 (that is actually my PyCharm hard-limit)

Reply:
For me it was 110. But I also do not mind 100. Both fits on my small 18 inch screen :D

Reply:
Yeah, let's not forget the scope of this bot, it should be a gentle reminder, not a barking dog (which I agree it sometimes is)